frame/support/procedural/src/pallet/parse/call.rs

```diff
@@ -93,6 +93,10 @@ impl syn::parse::Parse for FunctionAttr {
 			let call_index_content;
 			syn::parenthesized!(call_index_content in content);
 			let index = call_index_content.parse::<syn::LitInt>()?;
+			if !index.suffix().is_empty() {
+				let msg = format!("invalid suffix `{:?}` for number literal", index.suffix());
```

Comment:
```suggestion
				let msg = "Number literal must not have a suffix";
```

Reply:
Maybe you have a better idea for the error message? But there is no valid suffix, so we should not write "Invalid"

Reply:
Yes, you are right, I did not choose the correct word here.